src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/string.txt

```diff
@@ -280,4 +280,13 @@ true
 "ab"
 
 >> Concatenate(Text(1/0, "#.000"), "b", Text(7))
-#Error
\ No newline at end of file
+#Error
+
+>> $"Hello {"World!"}"
+"Hello World!"
+
+>> $"Hello {"World!"} 2"
+"Hello World! 2"
+
+>> $"One {$"Two {"Three"}"} Four"
+"One Two Three Four"
```

Comment:
Can you also add tests for the `{{` "escape"? Something like

    >> $"This string {{{"has"}}} curly braces"
    "This string {has} curly braces"

Reply:
Added